data/json/monsters.json

```diff
@@ -6718,6 +6775,46 @@
     "categories":["WILDLIFE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_rattlesnake_giant",
+    "name": "giant rattlesnake",
+    "species":"REPTILE",
+    "default_faction":"small_animal",
+    "symbol":"s",
+    "color":"brown",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":8,
+    "aggression":-10,
+    "morale":60,
+    "speed":100,
+    "melee_skill":6,
+    "melee_dice":1,
+    "melee_dice_sides":8,
+    "melee_cut":6,
+    "dodge":1,
+    "armor_bash":1,
+    "armor_cut":3,
+    "vision_day":30,
+    "vision_night":5,
+    "hp":48,
+    "death_function":"NORMAL",
+    "special_attacks":[
+        ["RATTLE", 6],
```

Comment:
Use the ````PARROT```` function instead of the ````RATTLE```` one and give it an appropriate entry in the speech.json file - you can adjust the volume in that file at the same time.